dbusmock/testcase.py

```diff
@@ -106,6 +106,8 @@ def __start_bus(cls, bus_type) -> None:
         (pid, addr) = cls.start_dbus(conf=str(conf))
         os.environ[f'DBUS_{bus_type.upper()}_BUS_ADDRESS'] = addr
         setattr(cls, f'{bus_type}_bus_pid', pid)
+        assert DBusTestCase.session_bus_pid is None
+        assert DBusTestCase.system_bus_pid is None
```

Comment:
See my previous comment. If we go back to actual global d-bus instances, these should be inverted to `is not`.

Reply:
Resolving this as "fixed" or rather "let's leave it", see [the thread in the previous comment](https://github.com/martinpitt/python-dbusmock/pull/184#discussion_r1388976334)